apps/hash-frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/latest-property-types-context.tsx

```diff
@@ -0,0 +1,43 @@
+import { PropertyType } from "@blockprotocol/graph";
```

Comment:
This is a move + expansion of `[entity-type-id].page/shared/latest-property-types-context.ts`